unified-prototype/unified-plugin/gradle/libs.versions.toml

```diff
@@ -1,6 +1,6 @@
 [versions]
 # Find latest version at: https://kotlinlang.org/docs/releases.html#release-details
-kotlin = "1.9.21"
+kotlin = "1.9.23"
```

Comment:
❗ Can we keep kotlin at version 1.9.21 for now?

This makes it easier to work with Now in Android.  Upgrading causes a bunch of issues that I don't think it's worth working through:

`ksp-1.9.21-1.0.16 is too old for kotlin-1.9.23. Please upgrade ksp or downgrade kotlin-gradle-plugin to 1.9.21.`

Reply:
We could alternatively allow NIA to declare that it needs Kotlin 1.9.21, or even better, properly model KSP.

In the meantime, however, I'll take this change out.

Reply:
Thanks.  Eventually, we should definitely upgrade NiA, but I abandoned my first attempt at that after it didn't seem like it would be quick.